extension/contentScript.js

```diff
@@ -11,122 +11,108 @@
 // @grant        none
 // ==/UserScript==
 
-(function() {
```

Comment:
I'd thought enclosing the whole thing in a function was important to stop the scope leaking out, but I could be wrong there. Tampermonkey includes it in the template for a new script.

Reply:
Tampermonkey uses a very old template. In any browser used today, there is no need to use IEEE functions.